docs/make.jl

```diff
@@ -0,0 +1,37 @@
+using Documenter, DataDrivenDiffEq
+
+makedocs(
+    sitename="DataDrivenDiffEq.jl",
+    authors="Julius Martensen, Chris Rackauckas",
+    modules=[DataDrivenDiffEq],
+    clean=true,doctest=false,
+    format = Documenter.HTML(#analytics = "UA-90474609-3",
+                             assets = ["assets/favicon.ico"],
+                             canonical="https://docs.sciml.ai/stable/"),
+    pages=[
+        "Home" => "index.md",
+        "basis.md",
```

Comment:
basis is more of a deeper reference. You want to grab the reader first, so the methods DMD SInDy etc. should come first, with the details about basis's afterwards for those now curious.